toolset/setup/linux/languages/pypy.sh

```diff
@@ -6,11 +6,11 @@ RETCODE=$(fw_exists ${IROOT}/pypy.installed)
   return 0; }
   
```

Comment:
This is a pedantic thing, but could we change this file (and every instance of `fw_depends` that points to `pypy` to be `pypy2` for clarity?